### PR TITLE
Rubocop layout rules fix

### PR DESCRIPTION
**Pull request by archanaserver**
For reference: https://github.com/theforeman/foreman/pull/9977

Fixed some of the Layout rules to satisfy theforeman-rubocop gem to v0.1.2

### CODE REVIEW

**Comment by archanaserver** [comment]
I want to drop the minitest commits from here, because seperated the work here https://github.com/theforeman/foreman/pull/9977, @ekohl i'm using `git rebase -i HEAD^n` command but that is not helping here.

File: .rubocop.yml [hunk 1]
```diff
@@ -98,3 +98,5 @@ Layout/EmptyLineBetweenDefs:
 Layout/FirstHashElementIndentation:
   Enabled: false
 
+Lint/Syntax:
```
Comment (ekohl) [inline .rubocop.yml]: This doesn't make sense to me. https://docs.rubocop.org/rubocop/cops_lint.html#lintsyntax only guarantees it's valid Ruby syntax. Why would you disable this?

Comment (archanaserver) [reply .rubocop.yml]: Sorry for the confusion, I was suppose to revert that change at the same time, was trying to look were from where the issue is coming and it was from here `test/controllers/api/v2/hosts_controller_test.rb` in 0d5eb62b5e7f7142e70106618a8a5b1e855c3a96

**Comment by archanaserver** [comment]
Fix Layout/ArgumentAlignment changes: https://github.com/theforeman/foreman/pull/10051

**Comment by ekohl** [comment]
I've converted this into a draft until https://github.com/theforeman/foreman/pull/10051 is merged.

**Pull request closed without merging**